content/workshops/openshift_4_101/lab1-welcome.md

```diff
@@ -13,6 +13,7 @@ We will be using the following terms throughout the workshop labs so here are so
 
 * Container - Your software wrapped in a complete filesystem containing everything it needs to run
 * Image - We are talking about docker images; read-only and used to create containers
+* Image Stream - An image stream comprises one or more Docker images identified by tags. 
```

Comment:
An image stream is comprised of one or more Docker images, which are identified with tags.